tribe-ext-settings-import-export.php

```diff
@@ -0,0 +1,370 @@
+
+			if ( version_compare( PHP_VERSION, $php_required_version, '<' ) ) {
+				if (
+					is_admin()
+					&& current_user_can( 'activate_plugins' )
+				) {
+					$message = '<p>';
+					$message .= sprintf( __( '%s requires PHP version %s or newer to work. Please contact your website host and inquire about updating PHP.', 'PLUGIN_TEXT_DOMAIN' ), $this->get_name(), $php_required_version );
+					$message .= sprintf( ' <a href="%1$s">%1$s</a>', 'https://wordpress.org/about/requirements/' );
+					$message .= '</p>';
+					tribe_notice( 'PLUGIN_TEXT_DOMAIN' . '-php-version', $message, [ 'type' => 'error' ] );
+				}
+				return false;
+			}
+			return true;
+		}
+
+		/**
+		 * Register the settings page
+		 */
+		function tribe_settings_menu() {
+			//add_options_page( __( 'Sample Settings Import and Export' ), __( 'Sample Settings Import and Export' ), 'manage_options', 'tribe-settings', 'tribe_settings_page' );
+			add_submenu_page( 'edit.php?post_type=tribe_events', __( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ), __( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ), 'manage_options', 'tribe_import_export', array( $this, 'tribe_sie_settings_page' ) );
+		}
+
+		/**
+		 * Render the settings page
+		 */
+		function tribe_sie_settings_page() {
+		    ?>
+			<div class="wrap">
+				<h2><?php _e( 'Settings Import / Export', 'PLUGIN_TEXT_DOMAIN' ); ?></h2>
+
+                <?php
+                // Success and error messages
+                if ( ! empty( $_GET['action'] ) ) {
+
+                    if ( $_GET['action'] == 'import_success' ) {
+	                    $msg = __( 'Settings imported.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-success ';
+                    }
+	                elseif ( $_GET['action'] == 'import_failed' ) {
+		                $msg = __( 'Settings imported.', 'PLUGIN_TEXT_DOMAIN' );
+		                $notice_class = 'notice-error ';
+	                }
+                    elseif ( $_GET['action'] == 'reset_success' ) {
+	                    $msg = __( 'Reset successful.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-success ';
+                    }
+                    elseif ( $_GET['action'] == 'reset_failed' ) {
+                        $msg = __( 'Settings reset failed.', 'PLUGIN_TEXT_DOMAIN' );
+                        $notice_class = 'notice-error ';
+                        }
+                    elseif ( $_GET['action'] == 'reset_no' ) {
+	                    $msg = __( 'Reset failed. Please enter "reset" in the text field to reset the settings.', 'PLUGIN_TEXT_DOMAIN' );
+	                    $notice_class = 'notice-error ';
+                    }
+
+	                echo '<div class="notice ' . $notice_class . ' is-dismissible"><p>' . $msg . '</p></div>';
+                } ?>
+
+				<div class="metabox-holder">
+					<div class="postbox">
+						<h3><span><?php _e( 'Export Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
```

Comment:
```suggestion
						<h3><span><?php esc_html__e( 'Export Settings', 'PLUGIN_TEXT_DOMAIN' ); ?></span></h3>
```

Looks like a string we could+should escape ... same note may apply elsewhere, too.